resources/install.rb

```diff
@@ -31,7 +31,9 @@
   package %w(curl tar)
 
   if new_resource.create_user
+    group 'hab'
```

Comment:
Put a space between the resources please. Otherwise they look like 1 at first glance.

Reply:
space added